src/crud/session.py

```
@@ -79,15 +101,24 @@ async def get_or_create_session(
         ConflictException: If we fail to get or create the session
     """
 
-    stmt = (
-        select(models.Session)
-        .where(models.Session.workspace_name == workspace_name)
-        .where(models.Session.name == session.name)
-    )
+    if not session.name:
+        raise ValueError("Session name must be provided")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)